src/cms/templates/language_tree/language_tree.html

```diff
@@ -52,6 +52,3 @@ <h2 class="heading font-normal mb-2">{% trans 'Language tree' %}</h2>
 </div>
 {% endblock %}
 
-{% block javascript %}
-<script src="{% static 'js/tree_drag_and_drop.js' %}"></script>
-{% endblock %}
```

Comment:
This JS was probably also removed by mistake.

Reply:
You are right, removing this js script was a fault, so i just amended these lines to my last commit and updated the django.po file with the translate script.